packages/mapping/ol/mapmodule/bundle.js

```diff
@@ -39,6 +39,13 @@ Oskari.clazz.define(
                     'type': 'text/javascript',
                     'src': '../../../../bundles/mapping/mapmodule/plugin/layers/LayersPlugin.ol.js'
                 },
+                /**
+                 * toggle layer coverage button
+                 */
+                {
+                    'type': 'text/javascript',
+                    'src': '../../../../bundles/mapping/mapmodule/plugin/layers/coveragetool/CoverageToolPlugin.js'
```

Comment:
Working towards removing/simplifying the packages/bundle.js: import in https://github.com/oskariorg/oskari-frontend/blob/master/bundles/mapping/mapmodule/AbstractMapModule.js#L35 instead. This also makes this available on the 3d mapmodule which might be broken without this import.